iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/NetworkDeserializerContext.java

```diff
@@ -35,6 +35,7 @@ public class NetworkDeserializerContext extends AbstractNetworkSerDeContext<Impo
     private ValidationLevel networkValidationLevel;
 
     private Set<String> ignoredEquipments = new HashSet<>();
+    private EnumSet<DeserializationEndTask.Step> alreadyProcessedEndTasksSteps = EnumSet.noneOf(DeserializationEndTask.Step.class);
```

Comment:
can be final, and renamed? as "already" and "processed" are carrying the same information twice
```suggestion
    private final EnumSet<DeserializationEndTask.Step> processedEndTasksSteps = EnumSet.noneOf(DeserializationEndTask.Step.class);
```